src/main/python/SentimentAnalyzer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+# pip install --upgrade azure-ai-textanalytics
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+_key = "adc96f39ac2a4918a1f462b5bf8674b8"
```

Comment:
It's usually not a good idea to hardcode api keys into code. We can either make this repo private and give the professor access when she needs to grade or there's a few other options (keyring, system property, etc.). If you don't mind this key being public, we could also just leave it as is. I just don't want to leave a security vulnerability out there in for someone outside this group to use your key.

Reply:
Agree, I will remove the key.

Reply:
What was your plan for removing this?